service/replication.go

```diff
@@ -745,7 +745,7 @@ func reprotect(ctx context.Context, localIsiConfig *IsilonClusterConfig, remoteI
 	// Create a new local policy based on previous remote policy's parameters
 	log.Info("Creating new local SyncIQ policy")
 	err = localIsiConfig.isiSvc.client.CreatePolicy(ctx, ppName, remotePolicy.JobDelay,
-		remotePolicy.TargetPath, remotePolicy.SourcePath, remoteIsiConfig.Endpoint, localIsiConfig.ReplicationCertificateID, true)
+		remotePolicy.TargetPath, remotePolicy.SourcePath, remoteIsiConfig.Endpoint, remoteIsiConfig.ReplicationCertificateID, true)
```

Comment:
Hi @tdawe,
May I know what was the error from _"reprotect: create protection policy on the local site failed %s", err.Error()_ ?

In PowerScale, we do have failback and reprotect operations. 
reprotect is done to re-establish the replication direction in reverse direction, once fail over is complete. Please refer this [section](https://dell.github.io/csm-docs/docs/replication/architecture/powerscale/#reprotect---set-original-target-as-new-source) of CSM Doc.

Reply:
Requesting the full log message since this was encountered by someone else testing PowerScale replication.

Reply:
@santhoshatdell , the error message:
```
reprotect: create protection policy on the local site failed Certificate "<uid-redacted>" not found in the synciq_peer certificate store.
```

Reply:
Approving based on the fact that createPolicy passed similar values during CreateVolume() and the e2e tests have passed.

Hope this is not Isilon clusters specific and their certificate related issues. Please double check that. Thanks!

Reply:
This has been tested across four unique Isilon clusters (2 without using SyncIQ Encryption and 2 with SyncIQ Encryption enabled). Reprotect action is successful in both environments.